packages/blocks/src/root-block/widgets/ai-panel/ai-panel.ts

```diff
@@ -24,7 +24,10 @@ import type {
 } from './components/index.js';
 
 export interface AffineAIPanelWidgetConfig {
-  answerRenderer: (answer: string) => TemplateResult<1>;
+  answerRenderer: (
+    answer: string,
+    state: AffineAIPanelState
```

Comment:
Will it influence action refactor? @pengx17

Reply:
maybe not. the provider does not care about how the answer is rendered